Sources/Track.swift

```diff
@@ -82,3 +81,18 @@ class Track {
         videoManager.sendHeartbeats()
     }
 }
+
+/// Utitlity to log sending event with a dump of the event.
+private func os_log_sending_event(_ event: Event, log: OSLog = .tracker, type: OSLogType = .debug) {
+    var eventDump = DumpOutput()
+    dump(event.toDict(), to: &eventDump)
+    os_log("Sending an event from Track:\n%@", log: log, type: type, eventDump.content)
```

Comment:
Note to self: Revisit the `os_log` implementation and consider using modern `Logger`, see [Create a Log Object to Organize Messages](https://developer.apple.com/documentation/os/generating-log-messages-from-your-code#Create-a-Log-Object-to-Organize-Messages), which supports string interpolation.